src/plugins/irc-events/error.js

```diff
@@ -2,21 +2,57 @@ var Msg = require("../../models/msg");
 
 module.exports = function(irc, network) {
 	var client = this;
-	irc.on("errors", function(data) {
+
+	// TODO: remove later
+	irc.on("irc_error", function(data) {
+		console.log("Got an irc_error");
+		irc.emit("error", data);
+	});
```

Comment:
It's a bit unclear currently as it may be a mistake in `irc-fw` that it has 2 separate events.